test/remove-markdown.js

```diff
@@ -110,7 +110,7 @@ describe('remove Markdown', function () {
         });
     });
 
-    it('should not remove greater than signs', function () {
+    it.skip('should not remove greater than signs', function () {
```

Comment:
shouldn't this test still pass?

Reply:
it will not pass because of `{ string: '> 100 > 0', expected: '100 > 0' }`

Reply:
Can you change the test to check the string `100 > 0`

Reply:
done